documentation/asciidoc/topics/yaml/grafana_datasource.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: grafanadatasource
+spec:
+  name: sso.yaml
```

Comment:
Not that it matters but why `sso`?

Reply:
It was copy & pasted from a deployment I was working on.

Reply:
@Crumby @ryanemerson Do we even need this field?

Reply:
@oraNod I'm not really sure. It's stating how the datasource file will named when injected into container.

Reply:
I think this name a bit misleading. We can use a name related to the Prometheus (source) and Grafana (destination) names combination, for example
https://developer.ibm.com/devpractices/devops/tutorials/custom-grafana-dashboards-from-pre-configured-prometheus-ocp-43-datasource/
https://zhimin-wen.medium.com/grafana-dashboard-in-ocp4-2-44468e5390d0

Reply:
@oraNod `Name` is marked as required in the 3.9.0 GrafanaDatasource CRD. I'd with something neutral like `datasource.yaml` or `prometheus-datasource.yaml`